realm/app-manager/src/launcher/oci.rs

```diff
@@ -0,0 +1,351 @@
+use std::collections::HashMap;
```

Comment:
There are some minor isses that can be discovered with clippy. Can you fix them?